guides/image-sources/provisioning-installer-bootdisk-fullhost.plantuml

```diff
@@ -0,0 +1,73 @@
+@startuml
+
+!include foreman.pstyle
+!$networkboot = 0
+
+title Installer-based provisioning with full-host boot disk on bare metal
+
+left header
+In the described scenario, Foreman does not manage DHCP.
+end header
+
+actor       User
+participant "Provisioned\nHost" as Host
+participant "Foreman" as Foreman
+participant "Foreman\nProxy" as Proxy
+participant DNS
+!if ($puppet)
+participant "Puppet\nserver" as Puppet
+!endif
+
+note over Host : Powered off
+
+== Create host in Foreman ==
+
+User -> Foreman : Create host
+User -> Foreman : Assign an MAC address
+Foreman -> Proxy : Create DNS records
+Proxy -> DNS : Forward DNS records
+Proxy -> Foreman : Report status of DNS
+Foreman -> User : Host is created\n(build mode enabled)
+
+User -> Foreman : Request to generate the full host boot-disk ISO
+group Templates [Boot disk iPXE - host, PXELinux, PXEGrub2]
+    Foreman -> Proxy : Render boot-loader configuration files
+end
+Foreman -> Proxy : Download kernel and initial RAM disk
+Foreman -> Foreman : Add the boot loader to ISO files
+Foreman -> Foreman : Build the full host boot-disk ISO
+Foreman -> User : Download the full host boot-disk ISO
+User -> User : Write the boot-disk ISO\nto an external storage device
+User -> Host : Configure to boot\nfrom the external storage device
+
+== Boot into OS installer ==
+
+User -> Host : Power on
+Host -> Host : Use the IP address from the boot disk
```

Comment:
Only if the static IP has been added during the host creation,  or we are in an unmanaged environment.
Otherwise, the machine will get an IP from the DHCP.

Reply:
I just saw the _header_ section, somehow I missed it in the picture.